moreores.lua

```diff
@@ -32,4 +32,15 @@ if bitchange.use_quartz and minetest.get_modpath("quartz") then
 		},
 		replacements = { {"default:pick_diamond", "default:pick_diamond"} }
 	})
-end
\ No newline at end of file
+end
+
+if bitchange.use_default_tin and minetest.get_modpath("default") then
```

Comment:
`minetest.get_modpath("default")` is always a string value (non-nil) since it's given by the dependencies.